docs/development.md

```diff
@@ -26,7 +25,10 @@ qdrouterd &
 ```shell
 cd <cloud-event-proxy repo>
 make build-plugins
-make run
+# Test with HTTP Transport
+go run cmd/main.go --transport-host="localhost:9043" --http-event-publishers="localhost:9043"
```

Comment:
consumer and publisher exposing to same service ?

Reply:
yes. This is a local test so the consumer and publisher share the same side car service.